src/libs/Navigation/shouldShowDeeplink/index.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'underscore';
+import CONST from '../../../CONST';
+/**
+ * @returns {Boolean}
+ */
+export default function shouldShowDeeplink() {
+    const currentUrl = window.location.pathname;
```

Comment:
This is kind of a weird situation platform-wise: this function only really makes sense on web (and will error if used on other platforms), but you're currently only using it on web. I think we should probably make this an index.web.js file for safety, but I also wouldn't object to it living inside the DeeplinkWrapper file since it is so specific to that use case.

Reply:
Alternative: this approach will work, but I think using `useRoute` (https://reactnavigation.org/docs/use-route) to compare react-navigation screen names instead of URL routes would be fine, and then it will work across all platforms (even though it has no use on other platforms).

Reply:
So I tried looking into using `useRoute` but the DeeplinkWrapper is outside of the navigation stack (maybe why they're using `window.location.pathname`?

Reply:
Okay yeah, that makes sense. There's still a way to make it route-based but it's more complicated and not worth it. I forgot pathname was being used before this.